### PR TITLE
Dev js first part tasks

### PR DESCRIPTION
**Pull request by KubaKlin**

### CODE REVIEW

File: tasks1.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+// 3
+const physicsGrade = 4
+const mathGrade = 3
+const englishGrade = 5
+
+const avarageGrade = (physicsGrade + mathGrade + englishGrade)/3;
+
+if (avarageGrade >=4 ) {
+  console.log('avarage grade is ' + avarageGrade + ' congrats!')
+}
```
Comment (Solnick) [inline tasks1.js:24]: > Otherwise, print "Try harder!"

You forgot about this

File: tasks1.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+const retirenmentAgeInPoland = 67;
+let yearsLeftToRetirenment;
+
+yearsLeftToRetirenment = retirenmentAgeInPoland - currentAge;
+
+console.log(yearsLeftToRetirenment);
+
+// 5
+const isPayingVAT = true;
+let VATRateInPoland = 0.23;
```
Comment (Solnick) [inline tasks1.js:37]: this can be `const`

File: tasks1.js [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+const incomeTaxRate = 0.12;
+let monthlyIncome = 3500;
```
Comment (Solnick) [inline tasks1.js:39]: this can be const

File: tasks1.js [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+const isSunny = false
+if (isSunny) {
+  console.log("Don't forget your sunglasses!")
+} else {
+  console.log("You might need an umbrella.");
+}
+
+// 7
+const currentMonth = 3;
+if ( currentMonth === 12 || currentMonth === 1 || currentMonth === 2 ) {
```
Comment (Solnick) [inline tasks1.js:60]:
```suggestion
if (currentMonth === 12 || currentMonth === 1 || currentMonth === 2) {
```
Please be consistent in code formatting

File: tasks1.js [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+  console.log("it's winter");
+} else if (currentMonth === 3 || currentMonth === 4 || currentMonth === 5) {
```
Comment (Solnick) [inline tasks1.js:62]:
```suggestion
} else if (currentMonth >= 3 || currentMonth <= 5) {
```
Let's make it simpler

File: tasks1.js [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+  console.log('Speed is fine');
+}
+
+// 16
+const ageVariable = 16;
+let isAdult = false;
+
+if(ageVariable >= 18) {
+  isAdult = true;
+}
```
Comment (Solnick) [inline tasks1.js:173]: Good! Can be done better ;)
```suggestion
const isAdult = ageVariable >= 18
```

File: tasks1.js [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+
+if(hasDrivingLicense) {
+  canDriveLegally = true;
+}
+
+if (isPersonAdult && canDriveLegally) {
+  console.log('This person can legally drive');
+} else {
+  console.log("This person can't legally drive");
+}
```
Comment (Solnick) [inline tasks1.js:195]: Almost
```suggestion
const canDriveLegally = isPersonAdult && hasDrivingLicense

if (canDriveLegally) {
  console.log('This person can legally drive');
} else {
  console.log("This person can't legally drive");
}
```

Comment (Solnick) [reply tasks1.js:195]: Maybe it can be simplified further…

**Review by Solnick — CHANGES_REQUESTED** [review]

File: tasks1.js [hunk 8]
```diff
@@ -0,0 +1,184 @@
...
+const personWeight = 80;
+const personHeight = 170;
+
+const personBMI = personWeight/(personHeight * personHeight);
+console.log(personBMI);
+
+// 9
+const age = 17
+if (age >= 18) {
+  console.log('a citizen can vote');
```
Comment (mwanago) [inline tasks1.js:80]: Use either `"` or `'`, but be consistent

File: index.html [hunk 9]
```diff
@@ -3,8 +3,9 @@
 <head>
   <meta charset="UTF-8">
   <title>Title</title>
+
```
Comment (mwanago) [inline index.html:6]: This empty line is not necessary

**Review by mwanago — APPROVED** [review]

**Pull request merged**